File: Linguagem C/1008.c
#include <stdio.h>

int main()
{
    int N,H;
    float V,S;

    scanf("%d",&N);
    scanf("%d",&H);
    scanf("%f",&V);

    S = H*V;

    printf("NUMBER = %d\n",N);
    printf("SALARY = U$ %.2f\n",S);

    return 0;
}

File: Linguagem GO/1008.go
package main
 
import (
    "fmt"
)
 
func main() {

	var numeroFuncionario int
	var horaTrabalhada, valorHora, salario float64
	
	fmt.Scanf("%d", &numeroFuncionario)
	fmt.Scanf("%f", &horaTrabalhada)
	fmt.Scanf("%f",&valorHora)
	
	salario = horaTrabalhada * valorHora
	
	fmt.Printf("NUMBER = %d\n", numeroFuncionario)
	fmt.Printf("SALARY = U$ %.2f\n", salario)
}
